applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -115,6 +115,8 @@ void BaseSolidElement::FinalizeSolutionStep( ProcessInfo& rCurrentProcessInfo )
     ConstitutiveLawOptions.Set(ConstitutiveLaw::COMPUTE_CONSTITUTIVE_TENSOR, false);
 
     Values.SetStrainVector(this_constitutive_variables.StrainVector);
+    Values.SetStressVector(this_constitutive_variables.StressVector);
```

Comment:
in the future please make a separate PR for things that are not related to this PR